tox.ini

```diff
@@ -60,3 +60,19 @@ commands_pre =
   uv pip install --reinstall .
 commands =
   mypy {posargs:.}
+
+[testenv:add_symlinks]
+changedir = {toxinidir}/src/phoenix
+allowlist_externals =
+  ln
+commands =
+  ln -sf ../../packages/phoenix-evals/src/phoenix/evals evals
+  ln -sf ../../packages/phoenix-otel/src/phoenix/otel otel
+
+[testenv:remove_symlinks]
```

Comment:
```suggestion
[testenv:link]
changedir = {toxinidir}/src/phoenix
allowlist_externals =
  ln
commands =
  ln -sf ../../packages/phoenix-evals/src/phoenix/evals evals
  ln -sf ../../packages/phoenix-otel/src/phoenix/otel otel

[testenv:unlink]
```

Suggestion

Reply:
that's a bit too vague because it would look like below

```
tox r -e link
```